src/extension.ts

```diff
@@ -30,7 +30,9 @@ export function activate(context: ExtensionContext) {
     let folders = workspace.workspaceFolders;
     context.subscriptions.push(new ProjectInitializer(!!versionInfo));
 
-    if (!folders || !folders.some((folder) => existsSync(path.join(folder.uri.fsPath, GAUGE_MANIFEST_FILE)))) return;
+    if (!folders || !folders.some((folder) => {
```

Comment:
No need of an anonymous method. `isGaugeFolder` has the same signature and can be passed directly to `folders.some`.